src/dlfcn.h

```diff
@@ -25,9 +25,18 @@
 #define DLFCN_H
 
 #ifdef __cplusplus
+#include <csetjmp>
+#define EXEC_CXX_FORK(JMPBUF)        std::setjmp(JMPBUF)
+#define EXIT_CXX_FORK(JMPBUF,STATUS) std::longjmp(JMPBUF,STATUS)

```

Comment:
Are these two macros used anywhere in the code?

Reply:
yes, in dlsym

Reply:
wait no, thought you were on about the other 2, added those for completeness

Reply:
Ack, as these are non-public macros, I think we can keep just the macro we actually use in the codebase.